notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>Add brief markdown here explaining the data/ingest. Just something simple like "This dataset contains images of flowers that have been serialized to TFRecords. Use the <code>tf.data</code> API to read and parse the data."  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Makes sense. Thanks!</p>